app/Services/ZugferdService.php

```
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Services;
+
+class ZugferdService
+{
+    public function __construct() {}
+
+    public function generateZugferdXml(string $xml): string {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$xml'. (undefined)

(UnusedFormalParameter)